src/utils/config/client/aad-config/simple-auth.provider.config.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { Field, ObjectType } from '@nestjs/graphql';
```

Comment:
it is in the aad-config directory?

Reply:
Good catch, fixed in 918ea36f025beb6911189a9fd093e34ebbc50b33